var/spack/repos/builtin/packages/py-biom-format/package.py

```diff
@@ -11,12 +11,17 @@ class PyBiomFormat(PythonPackage):
        a general-use format for representing biological sample by observation
        contingency tables."""
 
-    homepage = "https://pypi.python.org/pypi/biom-format/2.1.6"
-    url      = "https://pypi.io/packages/source/b/biom-format/biom-format-2.1.6.tar.gz"
+    homepage = "http://biom-format.org/"
+    url      = "https://github.com/biocore/biom-format/archive/2.1.10.tar.gz"
 
+    version('2.1.10', sha256='b63ce742dbe7d2353a48e004096645e9e2e4f7d53802e2c5d044f108b873a204')
+    version('2.1.9',  sha256='fce4ef925ef1a1594b16680f11c76e0ba245c0fda4286f3632a1bd4be36434c7')
+    version('2.1.8',  sha256='91cae2e11702c763eeb8a4e1db3cf120e7cdaae8cc38aaae7163be74e6cff77a')
+    version('2.1.7',  sha256='88b3ef16fd7d16f4e125f1e0d356aaad4c7c281ea3fb746c0f8dc26f3a5b6d4d')
     version('2.1.6', sha256='8eefc275a85cc937f6d6f408d91b7b45eae854cd5d1cbda411a3af51f5b49b0d')
 
     variant('h5py', default=True, description='For use with BIOM 2.0+ files')
+    variant('pyqi', default=False, description='For use with BIOM 2.1.6')
```

Comment:
If pyqi is required for certain versions, that should be listed as a dependency constraint rather than a variant